AdminPortal/src/styles/Login.module.css

```diff
@@ -0,0 +1,147 @@
+.outer {
+    display: flex;
+    height: 600px;
+
+    /*align-items: baseline;*/
+
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+
+    gap: 10.275rem;
+}
+
+.LAAKTAlogo {
+    
+    width: 445.31px;
+    height: 445.31px;
+    margin-top: -40px;
+    /*position: absolute;
+    left: 147px;
+    top: 206px;*/
+}
+
+.formContainer {
+    
+    width: 30vw;
+    height: 496.64px;
+
+    /*position: absolute;
+    left: 756.71px;
+    top: 247.62px;*/
+
+    /*margin-top: auto;*/
+
+    background: #FFFFFF;
+    box-shadow: 0px 8.32361px 41.6181px rgba(0, 0, 0, 0.25);
+    border-radius: 27.7454px;
+}
+
+.emailPassword {
+    display: flex;
+    flex-direction: column;
+}
+
+.emailInput {
+    box-sizing: border-box;
+    background: #FFFFFF;
+    color: black;
+    border: 1.38727px solid #C4C4C4;
+    width: 23vw;;
+    height: 38.84px;
+    margin: 10vh 3.5vw 0px 3.5vw;
+    padding-left: 10px;
+    border-radius: 5px;
+}
+
+.input::placeholder{
+    position: absolute;
+    left: 3.33%;
+    right: 85.61%;
+    top: 17.86%;
+    bottom: 22.93%;
+
+    /* Body + Buttons */
+
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 17px;
+
+    color: #101010;
+}
+
+.passwordInput {
+    box-sizing: border-box;
+    background: #FFFFFF;
+    color: black;
+    border: 1.38727px solid #C4C4C4;
+    width: 23vw;
+    height: 38.84px;
+    margin: 15.26px 3.5vw 0px 3.5vw;
+    padding-left: 10px;
+    border-radius: 5px;
```

Comment:
This is almost exactly the same as the styling for `.emailInput`. What you really want is a CSS class for "input boxes" in general in this application, or at least this page, and then if there's any special styling you want to apply to a particular input box then you can create another CSS class to go on top of that.

For instance, you could write something like
```css
.inputBox {
  border: 1px solid #c4c4c4;
  /* other styles... */
}

.redInputBox {
  border-color: #f00;
}
```
And in your component:
```javascript
<input className={styles.inputBox} />
<input className={`${styles.inputBox} ${styles.redInputBox}`} />
```

In this case, I don't see why these two input boxes should have different CSS styles. Figure out a way to make the margins work consistently (again, without crazy decimal values)

Reply:
have moved the common parts to another class textInputField as suggested

Reply:
I have moved all the components present inside the form into a separate div. Now I just align that div inside the white box with the shadow by calculating margin percentages (of width) through the figma design